elements/lisk-p2p/test/integration/advertise_address.ts

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
The test fits better into the functional test suite.

Reply:
I think its a configuration setting from the framework, so its an integration and also all other configuration tests are in integration. so it should be fine.